From d5eb8ded507b163a7488c64589db0e9e197b4fc8 Mon Sep 17 00:00:00 2001
From: Steven Moreland <smoreland@google.com>
Date: Thu, 25 Jan 2018 14:15:40 -0800
Subject: [PATCH 4/4] Make memtrack build with the VNDK.

When the vndk is enabled (BOARD_VNDK_VERSION=current),
vendor libraries must only link against vendor variants
in the build system. This also means that they receive
restricted sets of headers and that automatic global
headers are restricted.

Test: build memtrack with BOARD_VNDK_VERSION=current
Bug: 69526027
Change-Id: Ie3cb9ffb72931fbb0cc816f5158a649e09f7bb43
---
 msm8084/libmemtrack/memtrack_msm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/msm8084/libmemtrack/memtrack_msm.c b/msm8084/libmemtrack/memtrack_msm.c
index 4e4b78a3e..2f9301707 100644
--- a/msm8084/libmemtrack/memtrack_msm.c
+++ b/msm8084/libmemtrack/memtrack_msm.c
@@ -17,7 +17,7 @@
 #include <errno.h>
 #include <stdlib.h>
 #include <string.h>
-#include <utils/Log.h>
+#include <log/log.h>
 
 #include <hardware/memtrack.h>
 
-- 
2.17.1

